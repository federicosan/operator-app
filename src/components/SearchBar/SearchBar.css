.SearchBar {
  display: flex;
  width: 100%;  
  height: 50px;
}

.SearchBar input {
  flex-grow: 1;
  border: 0;
  outline: 0;
  font-size: 14px;
  padding: 0.25rem 1rem;
  color: #555;
}