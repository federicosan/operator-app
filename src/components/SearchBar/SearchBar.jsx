import React, { PropTypes } from 'react';
import { connect } from 'react-redux';

import PanelBar from '../PanelBar/PanelBar.jsx';

export class SearchBarComponent extends PanelBar {
  render () {
    return (
      <div className="row bar">
        <span>Search</span>
      </div>
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({
  dispatch,
});

const SearchBar = connect(
  mapStateToProps,
  mapDispatchToProps,
)(SearchBarComponent);

export default SearchBar;
