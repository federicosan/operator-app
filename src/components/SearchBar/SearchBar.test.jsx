import React from 'react';

import { shallow } from 'enzyme';

import SearchBar from './SearchBar.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('SearchBar', () => {
  it('matches snapshot', () => {
    const component = shallow(<SearchBar store={store} />);

    expect(component).toMatchSnapshot();
  });
});
