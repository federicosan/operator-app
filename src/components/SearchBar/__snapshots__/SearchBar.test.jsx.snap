exports[`SearchBar matches snapshot 1`] = `
<Connect(PanelBarComponent)
  dispatch={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
